### PR TITLE
PWN-7169 - local feature token model, new actions

### PR DESCRIPTION
**Pull request by NikolaiEmelianov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7169

## Description of Work

local feature token model, new actions

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/swap/jupiter/domain/model/SwapTokenModel.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    data class UserToken(
+        val token: Token.Active,
+    ) : SwapTokenModel {
+        override val decimals: Int = token.decimals
+        override val mintAddress: Base58String = token.mintAddress.toBase58Instance()
+    }
+
+    data class JupiterToken(
+        val token: JupiterSwapToken,
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/jupiter/domain/model/SwapTokenModel.kt:21]: token -> details

File: app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateAction.kt [hunk 2]
```diff
@@ -15,14 +16,14 @@ sealed class SwapStateAction {
...
 
     data class TokenAAmountChanged(
-        val tokenA: Token.Active,
-        val tokenB: Token,
-        val slippage: Double,
         val newAmount: BigDecimal
     ) : SwapStateAction()
+
+    object SwitchTokens : SwapStateAction()
+    object ConfirmTokenChanged : SwapStateAction()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateAction.kt:28]: SwapSuccess

File: app/src/main/java/org/p2p/wallet/swap/jupiter/domain/model/SwapTokenModel.kt (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/swap/jupiter/domain/model/SwapTokenModel.kt:21]: all the SwapTokens are JupiterToken
we can use our existent naming like Active / Other

Comment (NikolaiEmelianov) [reply app/src/main/java/org/p2p/wallet/swap/jupiter/domain/model/SwapTokenModel.kt:21]: SwapTokenModel is not only JupiterToken
It's wrapper over Token.Active as user token with balance information 
and wrapper over JupiterSwapToken

**Review by 1sakov — APPROVED** [review]

**Pull request merged**